tests/ccgbugs/tgc_bug.nim

```diff
@@ -1,5 +1,4 @@
 discard """
-matrix: "--gc:refc -d:fulldebug -d:smokeCycles"
```

Comment:
Why drop the defines while keeping them in `twrongrefcounting.nim`?

Reply:
I'm not entirely certain, but believe that it's an oversight -- I'm going to restore the defines here.